pkg/authorization/bootstrap/policy.go

```diff
@@ -25,9 +25,7 @@ import (
 // ClusterRoleBindings return default rolebindings to the default roles
 func clusterRoleBindings() []rbacv1.ClusterRoleBinding {
 	return []rbacv1.ClusterRoleBinding{
-		clusterRoleBindingCustomName(rbacv1helpers.NewClusterBinding("cluster-admin").Groups("system:kcp:clusterworkspace:edit").BindingOrDie(), "system:kcp:clusterworkspace:edit"),
-		clusterRoleBindingCustomName(rbacv1helpers.NewClusterBinding("view").Groups("system:kcp:clusterworkspace:view").BindingOrDie(), "system:kcp:clusterworkspace:view"),
-		clusterRoleBindingCustomName(rbacv1helpers.NewClusterBinding("admin").Groups("system:kcp:clusterworkspace:admin").BindingOrDie(), "system:kcp:clusterworkspace:admin"),
+		clusterRoleBindingCustomName(rbacv1helpers.NewClusterBinding("cluster-admin").Groups("system:kcp:clusterworkspace:admin").BindingOrDie(), "system:kcp:clusterworkspace:admin"),
```

Comment:
The name "cluster-admin" is not good. Would use `system:kcp:clusterworkspace:admin` as name, as the group.

Reply:
I'm not sure if I understand. The name "cluster-role" is used as a ClusterRole name that this ClusterRoleBinding binds to. Here's a copy-paste of that function:

```
func NewClusterBinding(clusterRoleName string) *ClusterRoleBindingBuilder {
	return &ClusterRoleBindingBuilder{
		ClusterRoleBinding: rbacv1.ClusterRoleBinding{
			ObjectMeta: metav1.ObjectMeta{Name: clusterRoleName},
			RoleRef: rbacv1.RoleRef{
				APIGroup: GroupName,
				Kind:     "ClusterRole",
				Name:     clusterRoleName,
			},
		},
	}
}
```

So the result of this whole function invocation will be ClusterRoleBinding with a name equal to `system:kcp:clusterworkspace:admin` that binds to "cluster-admin" ClusterRole. If I'm not mistaken, that seems correct and exactly what you asked for, right?

Reply:
ack, makes sense